### DISCUSSION TITLE
Does Threatmapper autorefresh it's vulnerability database ?

### DISCUSSION THREAD

**Discussion by rammanoj** [deepfence/ThreatMapper#1368] [Q&A] [answered]
There's no reference of it in documentation / any of the issues. Could someone help me with it ? 

If threat-mapper auto-refreshes the vulnerabilities, what are the sources from where is it refreshing from ?

**Comment by ibreakthecloud** [comment, marked as answer]
Deepfence has it's own ThreatIntel, which collects all the vulnerability from multiple sources and and refreshes the feed everyday. It is publically released in under 'Releases' in ThreatMapper repo.
